model/src/main/java/org/jboss/pnc/model/User.java

```diff
@@ -34,7 +34,7 @@
  * @author avibelli
  */
 @Entity
-@Table(uniqueConstraints = { @UniqueConstraint(name = "uk_user_email", columnNames = { "email" }),
+@Table(name = "UserTable", uniqueConstraints = { @UniqueConstraint(name = "uk_user_email", columnNames = { "email" }),
```

Comment:
I see it used to be called `Users` which actually sounds better than `UserTable`.